src/Raven.Client/ServerWide/Operations/Configuration/PutServerWideBackupConfigurationOperation.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Net.Http;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Documents.Session;
+using Raven.Client.Http;
+using Raven.Client.Json;
+using Sparrow.Json;
+
+namespace Raven.Client.ServerWide.Operations.Configuration
+{
+    public class PutServerWideBackupConfigurationOperation : IServerOperation
+    {
+        private readonly ServerWideBackupConfiguration _configuration;
+
+        public PutServerWideBackupConfigurationOperation(ServerWideBackupConfiguration configuration)
+        {
+            _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+        }
+
+        public RavenCommand GetCommand(DocumentConventions conventions, JsonOperationContext context)
+        {
+            return new PutServerWideClientConfigurationCommand(conventions, context, _configuration);
+        }
+
+        private class PutServerWideClientConfigurationCommand : RavenCommand
```

Comment:
You should probably rebase and inherit from `IRaftCommand`

Reply:
done